zoom_monitor.py

```diff
@@ -68,21 +75,26 @@ def _open_participants_list(self):
             for outer in self._driver.find_elements(
                     By.CLASS_NAME, "footer-button-base__img-layer"):
                 try:
+                    self.logger.debug(f"trying to find participants in {outer}")
```

Comment:
What does this log? I hope `{outer}` doesn't render as 30 lines long...

Reply:
Time to test! 😛

Reply:
(remember that you can test remotely: you can view the position of the servo through app.viam.com, and everything else is on your local machine which doesn't need to be in the office.)

Reply:
Yep, haha. That's what I'm doing now!

Reply:
Came out like this: `trying to find participants in <selenium.webdriver.remote.webelement.WebElement (session="057203abb89f95d5318e39e53edc7820", element="4DBD544D046DEF4052DEF968FDB495C1_element_15")>`

Reply:
Is that useful info? I imagine that, for every element, it will be identical except for the session ID and element ID, and those look like gibberish to me. When would this data be important?

Reply:
It more tells us that it's doing what we're expecting. Honestly, it could be "trying to find participant..." without the elements ids in there, but if we're logging it anyways, might as well. For example, the repeated elements helped me realize that we were redoing the check if the DOM wasn't set up correctly.